packages/scripts/src/generate-readme/generator.ts

```diff
@@ -152,6 +153,27 @@ export class ReadmeGenerator {
       //Note, not populating description, doesn't exist in framework adapters
       this.defaultEndpoint = adapter.defaultEndpoint ?? ''
     }
+
+    //TODO: Does the below still belong here? Came up in source branch when rebasing on develop
+    if (this.verbose) console.log(`${this.adapterPath}: Importing src/config/index.ts`)
+    this.defaultBaseUrl = configFile.DEFAULT_BASE_URL || configFile.DEFAULT_WS_API_ENDPOINT
```

Comment:
This was always there so don't mind if we consider this out of scope for this change but looks like we only set `this.defaultBaseUrl` with v2 standards. Should we add something in the v3 section for this too?

Reply:
Just saw Austin's comment above. This would only matter if the answer is that chunk should still be there